src/main/java/uk/gov/hmcts/cmc/claimstore/rpa/ClaimIssuedNotificationService.java

```diff
@@ -60,7 +61,7 @@ private EmailData prepareEmailData(Claim claim, List<PDF> documents) {
         EmailContent content = emailContentProvider.createContent(claim);
 
         EmailAttachment sealedClaimPdfAttachment = documents.stream()
-            .filter(document -> document.getFilename().contains("claim-form"))
+            .filter(document -> document.getClaimDocumentType() == SEALED_CLAIM)
```

Comment:
Suggestion - can we use equals() instead of ==. But Enum can be compared in both ways.